### PR TITLE
Update RuboCop to v0.51

### PR DESCRIPTION
**Pull request by AlexWayfer**
Take out `Timezone::Lookup::OptionSetter` from `class << self` due to false-positive RuboCop warning.

More info [here](https://github.com/bbatsov/rubocop/issues/5022).

### CODE REVIEW

**Comment by AlexWayfer** [comment]
> Any indication of when the OptionSetter related bug will be fixed in RuboCop? I would prefer to not work around bugs in RuboCop.

I made these changes because I didn't see such a declaration of private classes (in `class << self`) at all. 

And I don't really understand the difference, if there is any at all. It seems to me, without understanding it can be dangerous.

**Comment by panthomakos** [comment]
It looks like you'll need to drop Ruby `2.0` tests from Travis CI for this to work. Can you make that change to this PR?

**Comment by AlexWayfer** [comment]
@panthomakos, yes, I can. Changed commit pushed.

**Comment by panthomakos** [comment]
Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -5,8 +5,8 @@ branches:
 script:
   - bundle exec rake
 rvm:
-  - 2.0
   - 2.1
   - 2.2
   - 2.3
   - 2.4
+
```
Comment (AlexWayfer) [inline .travis.yml]: Oh, that's not good. That's [what I wrote about](https://github.com/panthomakos/timezone/pull/79#discussion_r152751873).

Comment (panthomakos) [reply .travis.yml]: This was caused by the merge conflict resolver in GitHub. I removed the trailing line on `master`.

**Pull request merged**